Source/ACE/Command/Handlers/DebugCommands.cs

```diff
@@ -825,8 +826,10 @@ public static void HandleWeapons(Session session, params string[] parameters)
             {
                 foreach (uint weenieId in weaponsTest)
                 {
-                    WorldObject loot = LootGenerationFactory.CreateTestWorldObject(session.Player, weenieId);
+                    AceObject lootAceObject = (AceObject)DatabaseManager.World.GetAceObjectByWeenie(weenieId).Clone(GuidManager.NewItemGuid().Full);
+                    WorldObject loot = new GenericObject(lootAceObject);
```

Comment:
[See this change in 482](https://github.com/ACEmulator/ACE/pull/482/commits/50df1daf9d03d82b8b8484c2e775380ad4418ee0)

Think it does what you're expecting, assuming your clone(guid) changes are also merged